api/renter.go

```diff
@@ -302,6 +292,12 @@ func (api *API) renterFilesHandler(w http.ResponseWriter, req *http.Request, _ h
 	})
 }
 
+// renterPricesHandler reports the expected costs of various actions given the
+// renter settings and the set of available hosts.
+func (api *API) renterPricesHandler(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	WriteJSON(w, api.renter.PriceEstimation())
+}
```

Comment:
Most API functions have their own struct defined. In this particular case I didn't see any reason that we might have extra fields that would not appear in the PriceEstimation struct. OTOH, if we eventually decide to change how something is encoded or have some compatibility departure, perhaps it is best that the struct get redeifined.

Actually, I think I will take a commit to properly define it as all the other functions do. =/ heh